readthedocs_theme/templates/includes/topnav.html

```diff
@@ -46,6 +46,7 @@
         <a class="item" href="{{ SITEURL }}/tools/sphinx">Sphinx</a>
         <a class="item" href="{{ SITEURL }}/tools/mkdocs">Mkdocs</a>
         <a class="item" href="{{ SITEURL }}/tools/jupyter-book">Jupyter Book</a>
+        <a class="item" href="{{ SITEURL }}/tools/custom">Any Tool</a>
```

Comment:
Casing, and "any tool" feels in conflict with the other menu options

```suggestion
        <a class="item" href="{{ SITEURL }}/tools/custom">Other tools</a>
```